priv/lang/alpaca.en_US.po

```diff
@@ -0,0 +1,84 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: alpaca\n"
+"POT-Creation-Date: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 1.8.7.1\n"
+
+msgid "builtin_type_arity_error %(num_expected) %(num_supplied)"
+msgstr ""
+"Wrong number of type parameters provided for builtin type \"%(type)\".\n"
+"Expected %(num_expected), but got %(num_supplied)."
+
+msgid "duplicate_definition %(id)"
+msgstr "Duplicate definition of \"%(id)\"."
+
+msgid "duplicate_type_definition %(id)"
+msgstr "Type \"%(id)\" has already been defined."
+
+msgid "function_not_exported %(mod) %(name)"
+msgstr "No function \"%(fun)\" exported from module \"%(mod)\"."
+
+msgid "incomplete_expression"
+msgstr "Unexpected end of expression."
+
+msgid "invalid_bin_qualifier %(qualifier)"
+msgstr ""
+"Invalid binary qualifier \"%(qualifier)\".\n"
+"Valid qualifiers are \\\"end\"\n"
+"-\"\\\", \\\"sign\\\", \\\"size\\\", \\\"type\\\" and \\\"unit\\\".\""
+
+msgid "invalid_bin_type %(type)"
+msgstr ""
+"Invalid binary part type \"%(type).\n"
+"Valid types are \"binary\", \"float, \"int\", and \"utf8\"."
+
+msgid "invalid_endianess %(endianess)"
+msgstr ""
+"Invalid endianess \"%(endianess)\".Did you mean \"big\", \"little\", or "
+"\"native\"?"
+
+msgid "invalid_fun_parameter"
+msgstr "Invalid pattern for function argument."
+
+msgid "invalid_top_level_construct"
+msgstr "This construct may not appear at the top level."
+
+msgid "module_rename %(old) %(new)."
+msgstr "Redefinition of module name from \"%(old)\" to \"%(new)\"."
+
+msgid "no_module"
+msgstr ""
+"No module name defined.\n"
+"You may define it like this: \"module foo\"."
+
+msgid "no_module %(mod)"
+msgstr "Cannot find any module named \"%(mod)\"."
+
+msgid "type_parameter_given_to_primitive_builtin_type %(type)"
+msgstr ""
+"Type parameter provided for built in type \"%(type)\", but none were "
+"expected."
+
+msgid "unexpected_token %(token)"
+msgstr "Symtax error before \"%(token)\"."
```

Comment:
Funny I managed to get this wrong both in po file *and* in the unit tests. Copy-paste error, I guess.